plugins/discovery/prometheus/utils.go

```diff
@@ -33,12 +33,16 @@ func init() {
 type prometheusEncoder struct{}
 
 func (e prometheusEncoder) Encode(ip, kind string, meta metav1.ObjectMeta, cfg interface{}) url.Values {
+	values := url.Values{}
 	rule := discovery.PluginConfig{}
 	discoveryType := "annotation"
 	if cfg != nil {
 		rule = cfg.(discovery.PluginConfig)
 		discoveryType = "rule"
+		values.Set("collectionInterval", rule.Collection.Interval.String())
```

Comment:
Can use `utils.Param(...)` since we will want to support these through annotations as well.